airflow/dag_processing/bundles/git.py

```diff
@@ -236,6 +238,8 @@ def view_url(self, version: str | None = None) -> str | None:
             return None
         if url.startswith("git@"):
             url = self._convert_git_ssh_url_to_https(url)
+        if url.endswith(".git"):
```

Comment:
Is this necessary? Have you tried to use it with .git at the end?

Reply:
I have tried to use it in [this](https://github.com/apache/airflow/blob/189d23c62ab56d4e0dd892a8319424168e56e510/tests/dag_processing/test_dag_bundles.py#L418).